universal-login-wallet/src/ui/react/Modals/Transfer/ModalTransfer.tsx

```diff
@@ -10,8 +10,8 @@ const ModalTransfer = () => {
   const [modal, setModal] = useState('transferAmount');
 
   const {walletService, sdk} = useServices();
-  const [transferDetails, setTransferDetails] = useState({currency: sdk.tokensDetailsStore.tokensDetails[0].symbol} as TransferDetails);
-
+  const [transferDetails, setTransferDetails] = useState({transferToken: sdk.tokensDetailsStore.tokensDetails[0].address} as TransferDetails);
```

Comment:
extract INTITIAL_TRANSFER_DETAILS?

Reply:
Now it takes the first element from tokensDetailsStore. Do you want to do some hardcoded value?